bundles/org.openhab.binding.androiddebugbridge/pom.xml

```
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.androiddebugbridge</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: Android Debug Bridge Binding</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.tananaev</groupId>
+      <artifactId>adblib</artifactId>
+      <version>1.2</version>
+    </dependency>
```

In order for the dependency to be embedded into the binding jar you need to mark it with the compile scope.
```suggestion
      <version>1.2</version>
      <scope>compile</scope>
    </dependency>
```